pkg/typedef/types_test.go

```diff
@@ -155,19 +155,19 @@ var prettytests = []struct {
 	},
 	{
 		typ: &typedef.BagType{
-			Kind:   "set",
-			Type:   typedef.TYPE_ASCII,
-			Frozen: false,
+			ComplexType: typedef.TYPE_SET,
+			Type:        typedef.TYPE_ASCII,
+			Frozen:      false,
 		},
 		query:    "SELECT * FROM tbl WHERE pk0=?",
 		values:   []interface{}{[]string{"a", "b"}},
 		expected: "SELECT * FROM tbl WHERE pk0={'a','b'}",
 	},
 	{
 		typ: &typedef.BagType{
-			Kind:   "list",
-			Type:   typedef.TYPE_ASCII,
-			Frozen: false,
+			ComplexType: "list",
```

Comment:
Please use typedef, instead of raw value

Reply:
Done